### PR TITLE
Add wipedevice function

### PR DESCRIPTION
**Pull request by wszdexdrf**

### CODE REVIEW

**Comment by danielabrozzoni** [comment]
Maybe we could manually run `test_wipe_device` in CI, as the last test run?

**Comment by wszdexdrf** [comment]
> Maybe we could manually run test_wipe_device in CI, as the last test run?

Should that be done in the code or by just adding another command in the CI? WDYT?

**Comment by wszdexdrf** [comment]
I was thinking of something else, but that is not possible here. It will have to be through CI commands.

File: src/lib.rs [hunk 1]
```diff
@@ -227,4 +228,20 @@ mod tests {
...
+    fn test_wipe_device() {
+        let devices = HWIClient::enumerate().unwrap();
+        for device in devices {
+            if device.device_type == "ledger" {
+                // Ledger devices don't support wipe
+                continue;
+            }
+            let client = HWIClient::get_client(&device, true, types::HWIChain::Test).unwrap();
+            client.wipe_device().unwrap();
+            break;
```
Comment (danielabrozzoni) [inline src/lib.rs:244]: You might as well just wipe every device here, without this break...

File: src/lib.rs [hunk 2]
```diff
@@ -227,4 +228,20 @@ mod tests {
...
     }
+
+    #[test]
+    #[serial]
+    #[ignore]
+    fn test_wipe_device() {
+        let devices = HWIClient::enumerate().unwrap();
+        let unsupported = ["ledger", "coldcard", "jade"];
+        for device in devices {
+            if unsupported.contains(&&device.device_type[..]) {
```
Comment (danielabrozzoni) [inline src/lib.rs:239]: You can write this as `&device.device_type.as_str()`

**Comment by danielabrozzoni** [comment]
ACK a17d127c6fe36b06dbd3138267cd36ab4d873546

**Comment by danielabrozzoni** [comment]
ACK 37337319c4e651f8908fc975b4aa6a8e311ed586

**Pull request merged**